lib/hexpm/accounts/users.ex

```diff
@@ -402,10 +402,11 @@ defmodule Hexpm.Accounts.Users do
   defp unset_email_flag_multi(multi, user, flag, audit: audit_data) do
     if old_email = Enum.find(user.emails, &Map.get(&1, flag)) do
       old_email_op = String.to_atom("old_#{flag}")
+      action = if flag == :public, do: :private, else: flag
 
       multi
       |> Multi.update(old_email_op, Email.toggle_flag(old_email, flag, false))
-      |> audit(audit_data, "email.#{flag}", {old_email, nil})
+      |> audit(audit_data, "email.#{action}", {old_email, nil})
```

Comment:
I think we should keep the audit log the same and instead change how we display it in the UI, so that if `new_email: nil` we say that the public email was removed/hidden.

Reply:
Addressed in https://github.com/hexpm/hexpm/pull/1038/commits/32e7621ce944e7985aef9f6f8b65b6885a7476d7